Bug Report: mqtt-to-postgresql in CrashLoop

### Expected Behavior

mqtt-to-postgres should not crash

### Current Behavior

it crashes in some edge cases and is not able to recover

### Relevant log output

```shell
2022-02-23T21:28:09.265Z	DEBUG	mqtt-to-postgresql/main.go:71	######################################################################################## Starting program..%!(EXTRA string=united-manufacturing-hub, string=factoryinsight, string=factoryinsight)
2022-02-23T21:28:09.265Z	INFO	mqtt-to-postgresql/main.go:73	This is mqtt-to-postgresql build date: 2022-02-18 08:32:56
2022-02-23T21:28:09.265Z	DEBUG	mqtt-to-postgresql/main.go:78	Setting up metrics%!(EXTRA string=/metrics, string=:2112)
2022-02-23T21:28:09.265Z	DEBUG	mqtt-to-postgresql/main.go:84	Setting up healthcheck
2022-02-23T21:28:09.265Z	DEBUG	mqtt-to-postgresql/main.go:100	Setting up raw queue
2022-02-23T21:28:09.332Z	DEBUG	mqtt-to-postgresql/main.go:103	Setting up MQTT
2022-02-23T21:28:09.333Z	INFO	mqtt-to-postgresql/mqtt.go:242	Running in Kubernetes mode%!(EXTRA string=united-manufacturing-hub-mqtttopostgresql-0, string=$share/MQTT_TO_POSTGRESQL/ia/#)
2022-02-23T21:28:09.333Z	DEBUG	mqtt-to-postgresql/mqtt.go:259	Broker configured%!(EXTRA string=tcp://united-manufacturing-hub-vernemq-local-service:1883, string=NO_CERT)
2022-02-23T21:28:09.338Z	INFO	mqtt-to-postgresql/mqtt.go:201	Connected to MQTT broker%!(EXTRA string=united-manufacturing-hub-mqtttopostgresql-0)
2022-02-23T21:28:09.348Z	INFO	mqtt-to-postgresql/mqtt.go:271	MQTT subscribed%!(EXTRA string=$share/MQTT_TO_POSTGRESQL/ia/#)
2022-02-23T21:28:09.348Z	DEBUG	mqtt-to-postgresql/main.go:108	Setting up redis
2022-02-23T21:25:12.082Z	DEBUG	mqtt-to-postgresql/mqtt.go:76	New MQTT message. Customer: REDACTED| Location: REDACTED| AssetId: bulovacnc | payloadType: processValue | Payload {"timestamp_ms":1645651512015,"REDACTED-1-4-AL1352-VVB001":41}
2022-02-23T21:25:12.082Z	DEBUG	mqtt-to-postgresql/processValueData.go:213	[ValueDataHandler]
2022-02-23T21:25:12.083Z	DEBUG	mqtt-to-postgresql/database.go:198	[GetAssetID] customerID: REDACTED, location: REDACTED, assetID: bulovacnc
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x593bf3]

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x593bf3]

goroutine 101 [running]:
github.com/beeker1121/goque.(*PriorityQueue).Enqueue(0x0, 0x0, {0xc0000fe000, 0x63, 0x70})
	/go/pkg/mod/github.com/beeker1121/goque@v2.1.0+incompatible/priority_queue.go:86 +0x73
main.ValueDataHandler.enqueueF64({0x0, 0x0, 0x15}, {0xc0000fe000, 0x63, 0x70}, 0x0)
	/build/cmd/mqtt-to-postgresql/processValueData.go:197 +0x65
main.ValueDataHandler.EnqueueMQTT({0x0, 0x0, 0x0}, {0xc00002a4e3, 0x5}, {0xc00002a4ee, 0x7}, {0xc00002a4f6, 0x0}, {0xc000316140, ...}, ...)
	/build/cmd/mqtt-to-postgresql/processValueData.go:278 +0x8a5
main.processMessage({0xc00002a4e3, 0x0}, {0xc00002a4ee, 0x7}, {0xc00002a4f6, 0x9}, {0xc00002a500, 0xc}, {0xc000316140, 0x40, ...})
	/build/cmd/mqtt-to-postgresql/mqtt.go:100 +0x136e
main.getOnMessageReceived.func1.1()
	/build/cmd/mqtt-to-postgresql/mqtt.go:192 +0x53
created by main.getOnMessageReceived.func1
	/build/cmd/mqtt-to-postgresql/mqtt.go:190 +0x208
```


### Steps to Reproduce

1. Prodution System, which has been running for some time
2. Stop mqtt-to-postgresql
3. Delete storage
4. Recreate mqtt-to-postgresql
5. VerneMQ will have some messages ready before mqtt-to-postgres will have created the queues

### Context (Environment)

VerneMQ was restarted therefore mqtt-to-postgresql must be restarted as well. During that it started crashing.

### Possible Solution

Could it be that the queues could not been set up before the first message arrives?

### Version

v0.8.x

### Anything else?

_No response_

--- a/golang/cmd/mqtt-to-postgresql/main.go
+++ b/golang/cmd/mqtt-to-postgresql/main.go
@@ -103,7 +103,6 @@ func main() {
 	zap.S().Debugf("Setting up MQTT")
 	podName := os.Getenv("MY_POD_NAME")
 	mqttTopic := os.Getenv("MQTT_TOPIC")
-	SetupMQTT(certificateName, mqttBrokerURL, mqttTopic, health, podName)
 
 	zap.S().Debugf("Setting up redis")
 	internal.InitCache(redisURI, redisURI2, redisURI3, redisPassword, redisDB, dryRun)
@@ -167,6 +166,9 @@ func main() {
 
 	//Only try to process old messages, once redis and pg are available !
 	storedRawMQTTHandler.Setup()
+
+	time.Sleep(1 * time.Second)
+	SetupMQTT(certificateName, mqttBrokerURL, mqttTopic, health, podName)
 
 	// Allow graceful shutdown
 	sigs := make(chan os.Signal, 1)

